wiki/Lua-Hyperspace-Module.md

```diff
@@ -1125,12 +1397,242 @@ function player_crew_health_boost()
+- [`Point`](#Point) `:GetFinalGoal()`
+- `int :GetIntegerHealth()`
+- [`Point`](#Point) `:GetLocation()`
+- `std::string :GetLongName()`
+- `float :GetMoveSpeed()`
+- `std::string :GetName()`
+- `bool :GetNewGoal()`
+- [`Point`](#Point) `:GetNextGoal()`
+- `int :GetRepairingId()`
+- `bool :GetResisted()`
+- [`Slot`](#Slot) `:GetSavedPosition()`
+- `int :GetSkillFromSystem(int systemId)`
+- `int :GetSkillLevel(int skillId)`
+- `float :GetSkillModifier(int skillId)`
+- [`int>`](#int>) `:GetSkillProgress(int skillId)`
+- `std::string :GetSkillTooltip(int skillId, int skillLevel, std::pair<int, int> progress, bool infoScreen)`
+- `std::string :GetTooltip()`
+- `void :IncreaseSkill(int skillId)`
+- `void :InitializeSkills()`
+- `bool :IsBusy()`
+- `bool :IsManningArtillery()`
+- `void :Kill(bool noClone)`
+- `void :MasterSkill(int skillId)`
+- `void :ModifyHealth(float health)`
+- `bool :MoveToRoom(int roomId, int slotId, bool forceMove)`
+- `bool :NeedFrozenLocation()`
+- `bool :NeedsSlot()`
+- `bool :OnInit()`
+- `void :OnRenderHealth()`
+- `void :OnRenderPath()`
+- `void :RenderSkillUpAnimation(Point pos)`
+- `bool :Repairing()`
+- `bool :RepairingFire()`
+- `bool :RepairingSystem()`
+- `void :Restart()`
+- `bool :RestorePosition()`
+- `bool :Sabotaging()`
+- `void :SavePosition()`
+- [`Pointf`](#Pointf) `:SelectSabotageTarget()`
+- `void :SetCloneReady(bool cloneReady)`
+- `void :SetCurrentShip(int shipId)`
+- `void :SetCurrentSystem(ShipSystem *sys)`
+- `void :SetCurrentTarget(CrewTarget *target, bool unk)`
+- `void :SetDamageBoost(float damageBoost)`
+- `void :SetDeathNumber(int deathNum)`
+- `void :SetFrozen(bool frozen)`
+- `void :SetFrozenLocation(bool frozenLocation)`
+- `void :SetHealthBoost(int healthBoost)`
+- `void :SetMedbay(float health)`
+- `void :SetMindControl(bool controlled)`
+- `void :SetName(TextString *name, bool force)`
+- `bool :SetPath(Path *path)`
+- `void :SetPosition(Point pos)`
+- `void :SetResisted(bool resisted)`
+- `void :SetRoom(int roomId)`
+- `void :SetRoomPath(int slotId, int roomId)`
+- `void :SetSavePosition(Slot position)`
+- `void :SetSex(bool male)`
+- `void :SetSkillProgress(int skillId, int skillLevel)`
+- `void :SetTask(CrewTask task)`
+- `void :StartRepair(Repairable *toRepair)`
+- `void :StartTeleport()`
+- `void :StartTeleportArrive()`
+- `void :StopRepairing()`
+- `void :UpdateHealth()`
+- `void :UpdateMovement()`
+- `bool :WithinRect(int x, int y, int w, int h)`
+- `void :constructor(CrewBlueprint blueprint, int shipId, bool intruder, CrewAnimation *animation)`
+
+### Fields
+- `int` `.iShipId`
+- `float` `.x`
+   - **read-only**
+- `float` `.y`
+   - **read-only**
+- `float` `.size`
+- `float` `.scale`
+- `float` `.goal_x`
+- `float` `.goal_y`
+- `int` `.width`
+- `int` `.height`
+- `pair<float, float> `.health`
+- `float` `.speed_x`
+- `float` `.speed_y`
+- [`Path`](#Path) `.path`
+- `bool` `.new_path`
+- `float` `.x_destination`
+- `float` `.y_destination`
+- [`Door*`](#Door) `.last_door`
```

Comment:
Do we need to specify pointers in these docs? There's quite a few instances of these in this update

Reply:
I feel like it's an important matter to know if we are dealing with a pointer to the class for modification purpose, like for the .choices thing that weren't giving back pointer and thus modification would have no effect.